apps/wallet-mobile/src/AppNavigator.tsx

```diff
@@ -201,17 +233,36 @@ const useHideScreenInAppSwitcher = () => {
   }, [])
 }
 
-type AuthAction = 'auth-with-pin' | 'auth-with-os' | 'request-new-pin' | 'first-run'
-const getAuthAction = (authOsEnabled: boolean, authSetting: AuthSetting): AuthAction => {
+type AuthAction =
```

Comment:
I believe this is not anymore about the authentication only Legal+Auth
```suggestion
type FirstAction =
```